api/controllers/game/lobby-data.js

```diff
@@ -1,21 +1,14 @@
-module.exports = function (req, res) {
+const { getLobbyDataByGame } = require('../../../utils/game-utils');
+
+module.exports = function returnLobbyData(req, res) {
   gameService
     .findGame({ gameId: req.session.game })
     .then(function sendResponse(game) {
-      const players = [];
-      if (game.players.length > 0) {
-        players.push({ username: game.players[0].username, pNum: 0 });
-        if (game.players.length > 1) {
-          players.push({ username: game.players[1].username, pNum: 1 });
-        }
+      const lobbyData = getLobbyDataByGame(game);
+      if (!lobbyData) {
+        throw new Error('Unable to get lobby data');
```

Comment:
Throw an error here instead so I don't need to throw one in the utility and the consuming function can handle it whatever way it wants. Also keeps your logic intact here.